docsets/Ramda_Chinese/docset.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Ramda Chinese",
+    "version": "0.1.0",
+    "archive": "Ramda_Chinese.tgz",
```

Comment:
Your file is called `Ramda_Chinese.tgz.txt`. It's supposed to be called `Ramda_Chinese.tgz`.